MobiFlightConnector.csproj

```diff
@@ -1690,7 +1690,7 @@
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <PropertyGroup>
-    <PostBuildEvent>if "$(ConfigurationName)" == "Release" ($(ProjectDir)Build\release.bat $(TargetDir) $(SolutionDir) 7.1.4)</PostBuildEvent>

```

Comment:
This isn't strictly required for this request, but I noticed it when working on the change. Doesn't seem like that version number needs to be there at all, the `release.bat` script only takes two parameters.

Reply:
you are right. this was like this in an early version of the script